solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -807,14 +856,34 @@ contract InterchainAccountRouter is Router {
             );
     }
 
-    /// @dev The calls represented by the commitment can only be executed once.
+    function callRemoteCommitReveal(
+        uint32 _destination,
+        bytes32 _commitment
+    ) public payable returns (bytes32 _commitmentMsgId, bytes32 _revealMsgId) {
+        bytes32 _router = routers(_destination);
+        bytes32 _ism = isms[_destination];
+
+        return
+            callRemoteCommitReveal(
+                _destination,
+                _router,
+                _ism,
+                bytes32(0),
+                bytes(""),
```

Comment:
is empty hook metadata really what we want?

Reply:
Updated to pass in a gas limit in the hook metadata